### ISSUE TITLE
question

### ISSUE THREAD

**Issue by yinshuangshuang621671** [nikhilbarhate99/PPO-PyTorch#63]
May I ask a question:  Why is the **with torch.no_grad():**  used in ppo.select_action ??

**Comment by GersiD** [comment]
If you don't use that then torch will accumulate gradients during the action selection process which can throw off the optimization step.
